services/motion/motion.go

```diff
@@ -91,3 +181,101 @@ func FromRobot(r robot.Robot, name string) (Service, error) {
 func FromDependencies(deps resource.Dependencies, name string) (Service, error) {
 	return resource.FromDependencies[Service](deps, Named(name))
 }
+
+// ToProto converts a PlanWithStatus to a *pb.PlanWithStatus.
+func (pws PlanWithStatus) ToProto() *pb.PlanWithStatus {
```

Comment:
Made these to / from proto methods public so that they could be used in the client tests.

Reply:
They shouldn't need to be. The tests should be in the same package right?

Reply:
They are not in the same package. The client tests are in `motion_test` package, which means it doesn't have access to motion private variables & functions. 

This is by design as that way the motion client tests show how our clients will actually interact with our package as consumers.